modules/dynamic-plugins/con-install-dynamic-plugin-helm.adoc

```diff
@@ -7,15 +7,25 @@
 [id="con-install-dynamic-plugin-helm_{context}"]
 = Installing dynamic plugins using the Helm chart
```

Comment:
Consider modularization. "Installing...", "add the following..." denotes a procedure module.